src/database/migrations/20191030205040-add-available-accommodation.js

```diff
@@ -0,0 +1,20 @@
+`use strict`;
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.addColumn(
+      'Accommodations',
+      'status',
```

Comment:
- Change the datatype to `Sequelize.BOOLEAN`, and the column name to `isAvailable`

Reply:
Thank you, I applied the changes